src/main/resources/db/migration/V8_16__update_parameter.sql

```diff
@@ -0,0 +1,21 @@
+drop table parameter;
+
+create table parameter
+(
+    errand_id      varchar(255),
+    id             varchar(255) not null,
+    parameters_key varchar(255),
```

Comment:
Add index on this column? 

(or is this not necessary if we get an index (fk) on errand_id?)